resources/views/checkout/partials/shipping-billing-fields.blade.php

```diff
@@ -23,12 +23,15 @@ class="sm:col-span-2"
         :required="Rapidez::config('customer/address/company_show', 'opt') == 'req'"
     />
     <x-rapidez-ct::input
-        label="Firstname"
         name="{{ $type }}_firstname"
+        label="Firstname"
```

Comment:
So currently this isn't possible to translate, keep it translatable just like currently in the core? https://github.com/rapidez/core/blob/master/resources/views/components/input.blade.php#L6